content/rs/references/client_references/client_ioredis.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: Using Redis with Node.js (ioredis)
+description:
+weight:
+alwaysopen: false
+categories: ["RS"]
+---
+In order to use Redis with Node.js, you need to install a Node.js Redis client.  In the following sections, we demonstrate the use of [ioredis](https://github.com/luin/ioredis), a community recommended Redis client for Node.js with support for Promises built in.  The other community recommended client for Node.js developers is [node_redis]({{< relref "./client_nodejs" >}}).
+
+Additional Node.js clients for Redis can be found under the [Node.js section](https://redis.io/clients#Node.js) of the [Redis Clients page](https://redis.io/clients).
+
+## Installing ioredis
+
+Complete instructions for installing ioredis can be found in the [README file](https://github.com/luin/ioredis/blob/master/README.md). To install ioredis, issue the following command:
+
+    npm install ioredis 
+
+## Connecting to Redis
+
+The following code creates a connection to Redis:
+
+    const Redis = require('ioredis');
+    const redis = new Redis({
+      host: 'hostname',
+      port: 6379,
+      password: 'password'
+    });
+
+Make sure to replace the following values with those for your Redis instance:
+
+- `'hostname'` should be the name of the host your database runs on
```

Comment:
s/should be/is/

We try not to use modal verbs except for can.

Reply:
Descriptions in a list use "-" to separate between the term and the description and end with a period if it is a complete sentence.